types.go

```diff
@@ -610,10 +616,32 @@ func (a Asset) MarshalJSON() (data []byte, err error) {
 	return json.Marshal(a.String())
 }
 
+// added 2022-11-02
+type RexInfo struct {
+	Version    uint32      `json:"version"`
+	Owner      AccountName `json:"owner"`
+	VoteStake  Asset       `json:"vote_stake"`
+	RexBalance Asset       `json:"rex_balance"`
+	MaturedRex uint64      `json:"matured_rex"`
+
+	// TODO: set the exact type
+	RexMaturities []interface{} `json:"rex_maturities"`
+}
+
+// added 2022-11-02
```

Comment:
Same for Go struct comment format

Reply:
https://github.com/eoscanada/eos-go/pull/191/commits/18d78b47dd7be8c506ce92f5422a4273eb0d1820